src/pages/Farm/index.tsx

```diff
@@ -29,18 +29,18 @@ import {
   TopBar,
   UpcomingPoolsWrapper,
 } from 'components/YieldPools/styleds'
+import { ZERO_ADDRESS } from 'constants/index'
 import { UPCOMING_POOLS } from 'constants/upcoming-pools'
 import { VERSION } from 'constants/v2'
-import { useTokens } from 'hooks/Tokens'
 import useMixpanel, { MIXPANEL_TYPE } from 'hooks/useMixpanel'
 import useParsedQueryString from 'hooks/useParsedQueryString'
 import { AppState } from 'state'
 import { useBlockNumber } from 'state/application/hooks'
+import { FarmUpdater, useElasticFarms } from 'state/farms/elastic/hooks'
 import { useFarmsData } from 'state/farms/hooks'
-import { useProMMFarms } from 'state/farms/promm/hooks'
 import { isInEnum } from 'utils/string'
 
-const Farms = () => {
+const Farm = () => {
```

Comment:
why

Reply:
change Yield to Farm.
Farm here is `Farm page`

Reply:
I mean why you change from `Farms` to `Farm`

Reply:
`Farm page` -> 1 page -> singular

Reply:
Farms page: page show list/table of farms
Farm page: page show details of 1 farm

Reply:
There are places you are using mix of `farm` and `farms`, like: 
```jsx
  <Route exact strict path="/farms" component={Farm} />
```

Reply:
it's `farms` not `Farm`. Nothing confuse here =))

Reply:
route is followed REST convention -> `farms`
component is a page -> `Farm`

Reply:
if you dont have any other concern, please resolve this

Reply:
I still not acceptable your idea =))